version/version.go

```diff
@@ -125,7 +125,9 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, fmt.Sprintf("error encoding JSON: %s", err), http.StatusInternalServerError)
 	} else {
 		w.WriteHeader(http.StatusOK)
-		w.Write(data)
+		if _, err := w.Write(data); err != nil {
+			http.Error(w, fmt.Sprintf("error writing the data to the connection: %s", err), http.StatusInternalServerError)
+		}
```

Comment:
How about:
```
if _, err := w.Write(data); err != nil {
    http.Error(w, fmt.Sprintf("error writing the data to the connection: %s", err), http.StatusInternalServerError)
} else {
    w.WriteHeader(http.StatusOK)
}
```

Reply:
Agree.

Reply:
You are right,
I fixed it.

Reply:
If that, you can combine the code like:
```
if err != nil {
    http.Error(w, fmt.Sprintf("error encoding JSON: %s", err), http.StatusInternalServerError)
} else if _, err := w.Write(data); err != nil {
    http.Error(w, fmt.Sprintf("error writing the data to the connection: %s", err), http.StatusInternalServerError)
} else {
    w.WriteHeader(http.StatusOK)
}
```
And in this way, you could reduce the ident? right? WDYT

Reply:
Thank you！
You are very good at the code style of golang.
I will change it.